### PR TITLE
Fix Postprocessing service

### PR DESCRIPTION
**Pull request by kobergj**
Makes it possible to use postprocessing service without `OCIS_INSECURE` flag

### CODE REVIEW

File: services/postprocessing/pkg/config/config.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type Events struct {
 	Endpoint string `yaml:"endpoint" env:"POSTPROCESSING_EVENTS_ENDPOINT" desc:"Endpoint of the event system."`
 	Cluster  string `yaml:"cluster" env:"POSTPROCESSING_EVENTS_CLUSTER" desc:"Cluster ID of the event system."`
 
-	TLSInsecure          bool   `yaml:"tls_insecure" env:"OCIS_INSECURE;SEARCH_EVENTS_TLS_INSECURE" desc:"Whether to verify the server TLS certificates."`
-	TLSRootCACertificate string `yaml:"tls_root_ca_certificate" env:"SEARCH_EVENTS_TLS_ROOT_CA_CERTIFICATE" desc:"The root CA certificate used to validate the server's TLS certificate. If provided SEARCH_EVENTS_TLS_INSECURE will be seen as false."`
+	TLSInsecure          bool   `yaml:"tls_insecure" env:"OCIS_INSECURE;POSTPROCESSING_EVENTS_TLS_INSECURE" desc:"Whether to verify the server TLS certificates."`
+	TLSRootCACertificate string `yaml:"tls_root_ca_certificate" env:"POSTPROCESSING_EVENTS_TLS_ROOT_CA_CERTIFICATE" desc:"The root CA certificate used to validate the server's TLS certificate. If provided SEARCH_EVENTS_TLS_INSECURE will be seen as false."`
```
Comment (micbar) [inline services/postprocessing/pkg/config/config.go:36]: Typo? `SEARCH_EVENTS...`?

Comment (kobergj) [reply services/postprocessing/pkg/config/config.go]: Yes. Typo from past me :smile:

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiGraph-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/17653/45/1) failed. Further test are cancelled...

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Web-Tests-ocis-smoke-ocis-storage-2</strong>](https://drone.owncloud.com/owncloud/ocis/17653/60/1) failed. Further test are cancelled...

**Review by micbar — APPROVED** [review]

**Pull request merged**